client/src/store/slices/users.slice.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const usersSlice = createSlice({
+  name: 'users',
+  initialState: {
+    usersList: [],
+    user: {},
+  },
+  reducers: {
+    initUsersList(state, action) {
+      state.usersList = action.payload.usersList;
+    },
+    initUser(state, action) {
+      state.user = action.payload.user;
+    },
+    changeUser(state, action) {
+      const { id } = action.payload;
+      const { user } = state.usersList.find((u) => u.id === id);
```

Comment:
why destruction user from find response ? 
it's not should be: `const user  = state.usersList.find((u) => u.id === id);`

Reply:
ok will change that, haven't noticed sir